obcsim/RS232.cpp

```diff
@@ -127,6 +127,15 @@ void RS_read(msp_link_t *lnk)
       }
       break;
 
+    case CMD_SEND_CITI_CALIB_CONF:
+        DEBUG_PRINT("CMD_SEND_CITI_CALIB_CONF received");
+        len = 4;
+        if (fill_commanddata(len)) {
+            DEBUG_PRINT("Invoking MSP_OP_SEND_CUBES_CITI_CALIB_CONF");
+            invoke_send(lnk, MSP_OP_SEND_CUBES_CITI_CALIB_CONF, commanddata, len, BYTES);
+        }
+        break;
```

Comment:
Please use two spaces for a TAB, so we keep to the (bad, I agree) style of the code in this file.